.github/wiki/Project-Configurations.md

```diff
@@ -0,0 +1,136 @@
+## Project Configurations
+
+This document presents in detail the set of configurations used in the iOS projects developed at Nimble. Included in this document:
```

Comment:
Small wording optimization: 
```suggestion
This document presents in detail the set of configurations used in the iOS projects developed at Nimble. Included in this document are:
```

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/268/commits/4ac43d42bd69de88234f85aee75cd8736fdfc56f